modules/create_timepoint/php/create_timepoint.class.inc

```diff
@@ -80,12 +80,23 @@ class Create_Timepoint extends \NDB_Form
             $site = \Site::singleton(intval($values['psc']));
         }
 
+        //Project
+        $projects = \Utility::getProjectList();
+        if (count($projects) == 1) {
+            //if there is only one project, autoselect first project from array of 1
+            //TODO: change this to array_key_first() when support is only PHP 7.3+
+            $project = \Project::getProjectFromID(array_keys($projects)[0]);
```

Comment:
I don't think you need a need function for that. 
```
$project = \Project::singleton(array_pop($projects));
```

Reply:
but I like get byID better...
ill switch it back

Reply:
I have to admit the dupicate function in the project class is not nice...